actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -134,13 +134,15 @@ def select_tag(name, option_tags = nil, options = {})
 
         if options.include?(:include_blank)
           include_blank = options.delete(:include_blank)
+          options_for_blank_options_tag = { value: '' }
 
           if include_blank == true
             include_blank = ''
+            options_for_blank_options_tag[:label] = ' '
```

Comment:
It's not specifying the value of the label attribute, though. It's saying that if there's no label attribute, then the _content of the option element_ may not be inter-element whitespace.